pkg/volume/csi/csi_mounter.go

```diff
@@ -153,6 +153,15 @@ func (c *csiMountMgr) SetUpAt(dir string, fsGroup *int64) error {
 
 	attribs := csiSource.VolumeAttributes
 
+	nodePublishSecrets := map[string]string{}
+	if csiSource.NodePublishSecretRef != nil {
+		nodePublishSecrets, err = getCredentialsFromSecret(c.k8s, csiSource.NodePublishSecretRef)
```

Comment:
nit: `"fetching NodePublishSecretRef failed for secret %s/%s failed: %v`

Reply:
Same as above.